app/controllers/concerns/parameters.rb

```diff
@@ -36,6 +36,19 @@ def new_relationship_ids(relationship)
       end
     end
 
+    def include_params
+      splitted = params[:include].split(',')
+      valids = serializer.relationships_to_serialize.keys.map(&:to_s)
+
+      if splitted & valids != splitted
```

Comment:
Can you please elaborate on what it does and how it compares the values?

Reply:
Intersection with a set equals with the original set.